libr/asm/p/asm_hexagon_gnu.c

```diff
@@ -16,7 +16,7 @@ hexagon_get_disassembler_from_mach(
   unsigned long big_p
 );
 static unsigned long Offset = 0;
-static char *buf_global = NULL;
+static RStrBuf *buf_global = NULL;
```

Comment:
see https://github.com/radare/radare2/pull/11606 for more fixes

Reply:
is this one? (buf_global = &op->buf_asm;)